src/phoenix/server/api/mutations/api_key_mutations.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
+from typing import Any, Dict, Optional
+
+import jwt
+import strawberry
+from sqlalchemy import insert, select
+from strawberry import UNSET
+from strawberry.types import Info
+
+from phoenix.db import models
+from phoenix.server.api.context import Context
+from phoenix.server.api.mutations.auth import HasSecret, IsAuthenticated
+from phoenix.server.api.queries import Query
+from phoenix.server.api.types.SystemApiKey import SystemApiKey
+
+
+@strawberry.type
+class CreateSystemApiKeyMutationPayload:
+    jwt: str
+    api_key: SystemApiKey
+    query: Query
+
+
+@strawberry.input
+class CreateApiKeyInput:
+    name: str
+    description: Optional[str] = UNSET
```

Comment:
We'll also need the expiry.

Reply:
Okay I can add that. I wasn't sure if you wanted system keys to not expire for some reason